package USER_ATTACK;

message CONTENTS
{
	message CHAR_ID_X_Y
	{
		optional int32 id = 1;
		optional int32 x = 2;
		optional int32 y = 3;
		optional int32 attckType = 4;
		optional int32 id_m = 5;
		optional int32 x_m = 6;
		optional int32 y_m = 7;
	}
	repeated CHAR_ID_X_Y data = 1;
}